demo/src/main/java/com/bluelinelabs/conductor/demo/MainActivity.java

```diff
@@ -12,22 +12,20 @@
 
 import butterknife.BindView;
 import butterknife.ButterKnife;
-import butterknife.Unbinder;
 
-public class MainActivity extends AppCompatActivity implements ActionBarProvider {
+public final class MainActivity extends AppCompatActivity implements ActionBarProvider {
 
     @BindView(R.id.toolbar) Toolbar toolbar;
     @BindView(R.id.controller_container) ViewGroup container;
 
     private Router router;
-    private Unbinder unbinder;
 
     @Override
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
 
         setContentView(R.layout.activity_main);
-        unbinder = ButterKnife.bind(this);
+        ButterKnife.bind(this);
```

Comment:
Imho, it doesn't hurt to release them manually, also shows off this ButterKnife feature (which is a bit new if I'm right)
Also helps to get used to using it in Controllers... Maybe a comment about not-being-required here would help. 

On the other hand it is also shown off in ButterKnifeController... :)